docs/rules/forbid-component-props.md

```diff
@@ -35,10 +35,14 @@ The following patterns are not considered warnings:
 
 ```js
 ...
-"forbid-component-props": [<enabled>, { "forbid": [<string>] }]
+"forbid-component-props": [<enabled>, { "forbid": [<string>], "includeDOM": [<boolean>]}]
 ...
 ```
 
 ### `forbid`
 
 An array of strings, with the names of props that are forbidden. The default value of this option is `['className', 'style']`.
+
+### `includeDOM`
```

Comment:
This seems like something that will vary per prop name. Perhaps the enhancement should be that `forbid` can have strings or objects, and if an object, it can be something like `{ "property": "foo", "includeDOM": true }`?